src/flag_gems/ops/layernorm.py

```diff
@@ -125,15 +125,15 @@ def layer_norm_backward_kernel(
         mask = row_mask and col_mask
         dy = tl.load(dY + cols[None, :], mask).to(tl.float32)
         x = tl.load(X + cols[None, :], mask).to(tl.float32)
-        x = tl.where(col_mask, x - mean, 0.0)
+        x = tl.where(mask, x - mean, 0.0)
         x_hat = x * rstd
         w = tl.load(W + cols, mask=cols < N).to(tl.float32)
         dx_hat = dy * w
         dx_part2 += dx_hat
         dx_part3 += dx_hat * x_hat
 
-    dx_2 = tl.sum(dx_part2)
-    dx_3 = tl.sum(dx_part3)
+    dx_2 = tl.sum(dx_part2, axis=1, keep_dims=True)
+    dx_3 = tl.sum(dx_part3, axis=1, keep_dims=True)
```

Comment:
it seems tl.sum in triton2.2 has no argument need keep_dims.

Reply:
The way to maintain dimensions has been modified. Or should we delete the BLOCK_ROW_SIZE parameter? That is, only support the case where it is equal to 1

Reply:
removing keep_dims is okay. I think dx_2 and dx_3 with 1-dimension could be automatically broadcast in the following computation.

Reply:
It misses a broadcasting in case of no keep_dims.
dx_2 = dx_2[:, None] suffices.